posts/2025/farmOS-3.4.0/index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: farmOS 3.4
+date: 2025-02-24
+author: Michael Stenta
+slug: 2025/farmOS-3.4
+categories:
+  - newsletter
+---
+
+# farmOS 3.4
+
+We are excited to announce the release of
+[farmOS 3.4](https://github.com/farmOS/farmOS/releases/tag/3.4.0)!
+This release includes a number of new features and improvements. Here are some
+of the highlights.
```

Comment:
I think this is a big improvement since it focuses attention on the main point of the post 3.4 - not a specific patch revision that might be assumed to be installable without further research/planning.

My only other thought would be that we should make the call-to-action at the bottom a bit clearer that the current patch version numbers can also be found in the changelog which is linked.

Reply:
Great. Good suggestion. Force-pushed the last commit with your suggested changes to the call to action.